tac/project-updates/2024/2024-Q4-Hyperledger-Indy.md

```diff
@@ -0,0 +1,145 @@
+[indy-besu]: https://github.com/hyperledger/indy-besu
+
+### **Client Tools**
+
+- [indy-vdr]
+- [indy-shared-rs] -- for AnonCreds, [Hyperledger AnonCreds] should be used
+- [indy-cli-rs] -- Indy CLI based on Indy VDR
+
+[indy-vdr]: https://github.com/hyperledger/indy-vdr
+[indy-shared-rs]: https://github.com/hyperledger/indy-shared-rs
+[indy-cli-rs]: https://github.com/hyperledger/indy-cli-rs
+[Hyperledger AnonCreds]: https://github.com/hyperledger/anoncreds-rs
+
+### **Specifications**
+
+- [did:indy Specification], [did:indy Specification source repository]
+- [did:indy Networks]
+- [Indy HIPE] (Hyperledger Indy Project Enhancements)
+- [indy-did-method]
+
+[did:indy Specification]: https://hyperledger.github.io/indy-did-method/
+[did:indy Specification source repository]: https://github.com/hyperledger/indy-did-method
+[did:indy Networks]: https://github.com/hyperledger/indy-did-networks
+[Indy HIPE]: https://github.com/hyperledger/indy-hipe
+[indy-did-method]: https://github.com/hyperledger/indy-did-method
+
+## Project Health
+
+The project had a further drop in contributor activity over the quarter. Per the
+[Indy Quarterly Activity Dashboard], in the months July through September, 2024
+there were just 18 commits (down about 80%) and 26 contributors (down about
+25%). Work was mainly on the [Indy Besu] initiative and in Indy monitoring
+tools. There is still a number of scaled deployments of Indy across the world,
+but all are running relatively smoothly, and there is little demand for new
+features in the existing Indy Node/Plenum code base.
+
+The Hyperledger Mentorship project has produced an [Indy Read Replica]
+implementation. We expect to have a
+[repository](https://github.com/rxbryan/indyread) containing a first cut of the
+functionality and documentation completed in November.
+
+[Indy Quarterly Activity Dashboard]: https://insights.lfx.linuxfoundation.org/foundation/lf-decentralized-trust/overview/github?project=indy&routedFrom=Github&bestPractice=false&dateFilters=Last%20Quarter&dateRange=2024-07-01%20to%202024-09-30&compare=PP&granularity=week&hideBots=true
+
+## Questions/Issues for the TOC
+
+None
+
+### Issues from previous reports
+
+#### **Diversity of Contributor Community**
+
+See the updated type of information in the appropriate section of this report.
+
+## Releases
+
+Releases since the last quarterly report:
+
+- indy-plenum - v1.14.0rc0 -- included in the last Quarterly report, but occurred in the period covered by this report.
+- indy-vdr - v0.4.3 -- included in the last Quarterly report, but occurred in the period covered by this report.
+
+## Overall Activity in the Past Quarter
+
+Work was done this quarter on the Ubuntu 22.04 version of Indy, with a PR
+submitted for Indy Node in July. However, that PR has not been approved/merged
+into the codebase. That has prevented the completion of a release candidate for
+the Indy on Ubuntu 22.04.
```

Comment:
Is the PR not being approved or merged due to lack of maintainers? Or is there another reason? Do we need to be concerned?

Reply:
@tkuhrt I tracked the PR [1] down and it does look a little concerning: Created 7 months ago, reviewers added 4 and 3 months ago, but no response to it so far. :/

[1] https://github.com/hyperledger/indy-node/pull/1870

Reply:
It is a cause for concern -- as noted in the report.  Lots of use still, but no demand for updates, and so no ongoing activities.  The review of that PR is a few solid days of focused work, and the other maintainers on the project have not been able to dedicate time for that review.

Reply:
Should we consider moving this project to Dormant?